README.md

```diff
@@ -10,6 +10,7 @@
   <p>
     <!-- <a href="https://crates.io/crates/magical"><img alt="Crate Info" src="https://img.shields.io/crates/v/magical.svg"/></a> -->
     <a href="https://github.com/bitcoindevkit/bdk/actions?query=workflow%3A%22Continuous+integration%22"><img alt="Continuous integration Status" src="https://github.com/bitcoindevkit/bdk/workflows/Continuous%20integration/badge.svg"></a>
+    <a href="https://codecov.io/gh/bitcoindevkit/bdk"><img src="https://codecov.io/gh/bitcoindevkit/bdk/branch/master/graph/badge.svg?token=5EG2G7GAM3"/></a>
```

Comment:
Is that token necessary? If I try to open the url without it, it still seems to work fine

Reply:
This is just the URL provided on the settings page, it's not the same token used for uploading new test runs. But if it works without it then I'll take it off an we can try it that way. Worst case I just need to put it back on.

Reply:
No problem then, I was afraid this was something related to a specific account/org id. If it was like that in the settings there's probably a valid reason, better to keep it i guess.